server/server_test.go

```diff
@@ -0,0 +1,73 @@
+package server
+
+import (
+	"context"
+	"github.com/brianvoe/gofakeit/v5"
```

Comment:
Ah! I was wanting something like this last week, this is super helpful for random values